e2e/test/scenarios/visualizations-tabular/drillthroughs/combine-column.cy.spec.js

```diff
@@ -86,5 +86,31 @@ describeWithSnowplow(
         database_id: SAMPLE_DB_ID,
       });
     });
+
+    it("should handle duplicate column names", () => {
+      openPeopleTable({ limit: 3, mode: "notebook" });
+
+      cy.findByLabelText("Pick columns").click();
+      popover().within(() => {
+        cy.findByText("Select none").click();
+        cy.findByLabelText("Email").click();
+      });
+
+      cy.findByLabelText("Pick columns").click();
+      cy.button("Visualize").click();
```

Comment:
We usually try to avoid using UI for the test setup - it can save a few seconds every time the test runs.

Something like this:

```suggestion
      createQuestion(
        {
          query: {
            "source-table": PRODUCTS_ID,
            fields: [["field", PRODUCTS.EMAIL, { "base-type": "type/Text" }]],
            limit: 3,
          }
        },
        { visitQuestion: true },
      );
```